### DISCUSSION TITLE
Uninstalling?

### DISCUSSION THREAD

**Discussion by ILikeProgramming101** [Klocman/Bulk-Crap-Uninstaller#430] [Q&A] [answered]
How do I uninstall it?

**Comment by Klocman** [comment, marked as answer]
Go to the `Help > Uninstall BCUninstaller` menu (assuming you installed it with the installer, otherwise just remove the BCUninstaller folder).

**Reply by NCPereira** [reply]
I downloaded the 5.5_setup.exe instead of the 5.5_portable.zip but during the installation I choose the "portable" option instead of the full installation. Is everything still contained in the single BCU folder? Can I remove everything simply by deleting this folder? Are any registry entries created for BCU at all?
I apologise for the dumb questions. Thanks in advance.

**Reply by Klocman** [reply]
Yes, it's the same as downloading the zip version.
